R/EpiStats.R

```diff
@@ -574,6 +565,16 @@ build_epistats <- function(geog.lvl = NULL,
   return(out)
 }
 
+#' for a race `n` the combo for `race1 == race2 ==n` the combo is is `2n - 1`
+#' for `race1 == n && race2 != n` the combo is `2n`
+#' @param race1 a vector of race index for the index
+#' @param race2 a vector of race index for the partners
+#' @return a vector of the same size with the race combos
+make_race_combo <- function(race1, race2) {
```

Comment:
can you add the `#' @export` flag here so it can be used within EpiModelHIV-p as well.

This should also make the documentation pass

```r
#' Generate Unique Race Combos Indentifiers for Partnerships
#'
#' The combo for `race1 == race2 == n` the combo is is `2n - 1`  
#' for `race1 == n && race1 != race2` the combo is `2n`
#'
#' @param race1 a vector of race index for the index
#' @param race2 a vector of race index for the partners
#' @return a vector of the same size with the race combos

#' @export 
```

you will have to run `devtools::document()` again

Reply:
@AdrienLeGuillou - Done!